packages/opentelemetry/lib/tracing/tracer.js

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable no-unused-vars */
+
+import TraceAPI, {SpanStatusCode, SpanOptions, setSpan } from '@opentelemetry/api';
+
+import {Span } from '@opentelemetry/tracing';
+
+class Tracer {
+
+  constructor (tracerProviderInstance, name, options = {}) {
+    this._tracerProvider = tracerProviderInstance;
+    const {version } = options;
+    this._tracer = TraceAPI.trace.getTracer(name, version);
+  }
+
+  /**
+   * creates a span object with optional parent span and optional span options
+   * @param {string} name  name of the span
+   * @param {Span} parentSpan (Optional) parentSpan to get context from
```

Comment:
`{?Span}` same below for nullable values

Reply:
updated